internal/users/tests/tests.go

```diff
@@ -1,12 +1,15 @@
 // Package tests export users test functionalities used by other packages to change cmdline and group file.
 package tests
 
+//nolint:gci // We import unsafe as it is needed for go:linkname, but this confuses gci so we should ignore it.
```

Comment:
I think the comment should be better: it’s the extra space before the nolint unsafe which confuses gci.